core/src/test/scala/integration/kafka/api/AdminClientIntegrationTest.scala

```diff
@@ -743,8 +742,8 @@ class AdminClientIntegrationTest extends IntegrationTestHarness with Logging {
 
     sendRecords(producers.head, 10, topicPartition)
     var result = client.deleteRecords(Map(topicPartition -> RecordsToDelete.beforeOffset(5L)).asJava)
-    var lowWatermark = result.lowWatermarks().get(topicPartition).get().lowWatermark()
-    assertEquals(5L, lowWatermark)
+    var lowWatermark = Option(result.lowWatermarks.get(topicPartition).get.lowWatermark)
+    assertTrue(lowWatermark.contains(5L))
```

Comment:
It's a bit better to use `assertEquald(Some(5), lowWatermark)` so that you get a good error message.

Reply:
Ok, I'll change that  once Jason has chimed in so that I can address his comments as well if he has any.